docs/language/accounts.mdx

```diff
@@ -14,10 +14,30 @@ Every account can be accessed through two types:
       let storageUsed: UInt64
       let storageCapacity: UInt64
 
+      // Keys
+      let keys: AuthAccount.Keys
```

Comment:
Should this be defined in both `PublicAccount` and `AuthAccount`?

Reply:
Yeah it is defined in both: (https://github.com/onflow/cadence/pull/635/files#diff-922d8e6ca000c087b0eba7a1886b7bccde23a0457bf3412dda16121293a372aeR63) due to the same reason as above.